Dockerfile

```diff
@@ -1,9 +1,66 @@
-FROM ubuntu:latest
-RUN apt-get update
-RUN apt-get install -y python3 python3-pip
-RUN pip3 install kubernetes
-RUN pip3 install PTable
```

Comment:
Since PTable was already being used, I decided to replace the imperative installation approach for a more declarative approach i.e.: listing PTable along with its version in requirements.txt.

> PrettyTable was [last updated in 2014](https://code.google.com/archive/p/prettytable/source/default/commits) and the PTable project [homepage](https://pypi.org/project/PTable/) mentions being forked from PrettyTable :wink:.